NitroxClient/GameLogic/Spawning/WorldEntitySpawner.cs

```diff
@@ -31,24 +30,28 @@ public WorldEntitySpawner(EntityMetadataManager entityMetadataManager, PlayerMan
 
     protected override IEnumerator SpawnAsync(WorldEntity entity, TaskResult<Optional<GameObject>> result)
     {
-        EntityCell cellRoot = EnsureCell(entity);
-        if (cellRoot == null)
+        bool found = TryFindAwakeParentCell(entity, out EntityCell parentCell);
+        if (found)
         {
-            // Error logging is done in EnsureCell
-            return null;
+            parentCell.EnsureRoot();
         }
-
         Optional<GameObject> parent = (entity.ParentId != null) ? NitroxEntity.GetObjectFrom(entity.ParentId) : Optional.Empty;
 
+        // No place to spawn the entity
+        if (!found && !parent.HasValue)
+        {
+            return null;
```

Comment:
We should add a debug log here to see what entities this affects and if it has unintended consequences. Or do you think that spam the log too much?

Reply:
It will definitely spam too much. I've tried looking at which entities actually go here and it was always entities out of range. My concern here is: if we have no parent nor cell, then we might break the cell system by forcing one entity to go in a cell that isn't awake at the time